library/src/scripts/headers/titleBarStyles.story.tsx

```diff
@@ -0,0 +1,685 @@
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import { StoryHeading } from "@library/storybook/StoryHeading";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import { MemoryRouter } from "react-router";
+import TitleBar from "@library/headers/TitleBar";
+import { Provider } from "react-redux";
+import getStore from "@library/redux/getStore";
+import { testStoreState } from "@library/__tests__/testStoreState";
+import { LoadStatus } from "@library/@types/api/core";
+import { IMe } from "@library/@types/api/users";
+import PageContext from "@library/routing/PagesContext";
+import { loadTranslations } from "@vanilla/i18n";
+import { TitleBarDeviceProvider } from "@library/layout/TitleBarContext";
+import { storyWithConfig } from "@library/storybook/StoryContext";
+import { color, linearGradient } from "csx";
+import { BannerContextProvider } from "@library/banner/BannerContext";
+import { colorOut } from "@library/styles/styleHelpersColors";
+import { BorderType } from "@library/styles/styleHelpersBorders";
+
+const localLogoUrl = require("./titleBarStoryLogo.png");
+
+loadTranslations({});
+
+export default {
+    title: "Headers",
+    parameters: {
+        chromatic: {
+            viewports: [1400, 400],
+        },
+    },
+};
+
+const makeMockRegisterUser: IMe = {
+    name: "Neena",
+    userID: 1,
+    permissions: [],
+    isAdmin: true,
+    photoUrl: "",
+    dateLastActive: "",
+    countUnreadNotifications: 1,
+};
+
+const initialState = testStoreState({
+    users: {
+        current: {
+            status: LoadStatus.SUCCESS,
+            data: makeMockRegisterUser,
+        },
+    },
+    theme: {
+        assets: {
+            data: {
+                logo: {
+                    type: "image",
+                    url: localLogoUrl as string,
+                },
+            },
+        },
+    },
+});
+
+function StoryTitleBar(props: { title: string }) {
+    return (
+        <PageContext.Provider
+            value={{
+                pages: {
+                    // search?: IPageLoader;
+                    // drafts?: IPageLoader;
+                },
+            }}
+        >
+            <MemoryRouter>
+                <BannerContextProvider>
+                    <TitleBarDeviceProvider>
+                        <StoryHeading depth={2}>{props.title}</StoryHeading>
+                        <TitleBar useMobileBackButton={false} isFixed={true} />
+
+                        <p>Scrollable content</p>
```

Comment:
This feels a bit bloated to have in line by line. Maybe we could replace it with:

```
{Array.from(new Array(100)).map(i => <p key={i}>Scrollable Content</p>)}
```